README.md

```diff
@@ -16,16 +16,28 @@ If bundler is not being used to manage dependencies, install the gem by executin
 ## Usage
 
 ```ruby
-directory = OCFL::Object::Directory.new(object_root: '/files/[object_root]')
-directory.exists?
+storage_root = OCFL::StorageRoot.new(base_directory: '/files', layout: OCFL::Layouts::DruidTree.new)
+storage_root.exists?
 # => false
-builder = OCFL::Object::DirectoryBuilder.new(object_root: 'spec/abc123', id: 'http://example.com/abc123')
-builder.copy_file('sig/ocfl.rbs')
+storage_root.valid?
+# => false
+
+storage_root.save
+storage_root.exists?
+# => true
+storage_root.valid?
+# => true
+
+object = storage_root.object('bc123df4567') # returns an instance of `OCFL::Object` (what used to be `OCFL::Object::Directory`)
```

Comment:
This will use the storage root's configured layout to figure out where to place the object and what ID to use for the new object.